### PR TITLE
fix(tests): force curent window closure

### PR DESCRIPTION
**Pull request by vladikoff**
Trying to fix https://github.com/mozilla/fxa-content-server/issues/2609

The main issue seems to be with switching window contexts. going to run a few ci runs on circle ci to see if it helps.

### CODE REVIEW

**Comment by pdehaan** [comment]
```
tests/functional/sync_reset_password.js
  157:27  error  Missing semicolon  semi
✖ 1 problem (1 error, 0 warnings)
```

File: tests/functional/sync_reset_password.js [hunk 1]
```diff
@@ -151,9 +152,10 @@ define([
         .end()
 
         .closeCurrentWindow()
+        .end()
+
         // switch to the original window
         .switchToWindow('')
-        .end();
```
Comment (pdehaan) [inline tests/functional/sync_reset_password.js]: This seems to be where ESLint is crying. Need a ";" at the end of line 158, `.switchToWindow('');`, or somethin'.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f696501881bf396d4ba191b849ef7e415293f909
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: bdd34bb390155c59cb5b3386d80ca0f193c9350a
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by pdehaan** [comment]
@vladikoff Typo in commit message, should be "current".

**Pull request closed without merging**